### PR TITLE
libmysqlclient, fix broken link, add debug package, move cmd

### PR DESCRIPTION
**Pull request by Begasus**

### CODE REVIEW

File: dev-db/libmysqlclient/libmysqlclient-6.1.6.recipe [hunk 1]
```diff
@@ -15,26 +15,36 @@ PATCHES="libmysqlclient-6.1.6.patchset"
...
+	devel:libmysqlclient_r$secondaryArchSuffix = $libVersionCompat
 	"
 REQUIRES_devel="
-	libmysqlclient$secondaryArchSuffix == 18.3.0
+	libmysqlclient$secondaryArchSuffix == $portVersion base
+	"
+
+PROVIDES_tools="
+	libmysqlclient${secondaryArchSuffix}_tools = $portVersion
+	cmd:my_print_defaults$secondaryArchSuffix
```
Comment (Begasus) [inline dev-db/libmysqlclient/libmysqlclient-6.1.6.recipe]: Not sure if this should be in tools or in the devel package, comments welcome :)

**Comment by Begasus** [comment]
Build checked on _x86 and x86_64

File: dev-db/libmysqlclient/libmysqlclient-6.1.6.recipe (hunk 1 above)
Comment (korli) [reply dev-db/libmysqlclient/libmysqlclient-6.1.6.recipe]: no, only mysql_config should be in _devel.

**Comment by Begasus** [comment]
Should be good to go, checked with LibreOffice that installs and runs OK with these chanes

File: dev-db/libmysqlclient/libmysqlclient-6.1.6.recipe [hunk 2]
```diff
@@ -15,26 +15,36 @@ PATCHES="libmysqlclient-6.1.6.patchset"
 ARCHITECTURES="!x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86"
 
+libVersion="18.3.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
```
Comment (korli) [inline dev-db/libmysqlclient/libmysqlclient-6.1.6.recipe]: is libVersionCompat then 18?

Comment (Begasus) [reply dev-db/libmysqlclient/libmysqlclient-6.1.6.recipe]: ah, should be 18.3? will look into it

Comment (Begasus) [reply dev-db/libmysqlclient/libmysqlclient-6.1.6.recipe]: Should be ok now

**Pull request merged**